packages/web-components/fast-components/src/tabs/index.ts

```diff
@@ -17,8 +17,8 @@ import { TabsStyles as styles } from "./tabs.styles";
     styles,
 })
 export class FASTTabs extends Tabs {}
-export * from "./tab";
-export * from "./tab-panel";
+export * from "../tab";
+export * from "../tab-panel";
```

Comment:
I know these were previously being exported from this file, but now that we've gone to a flat structure, does this still make sense? I would expect that we might just export these from their own folders and export them directly from the root.

Reply:
@rimildeyjsr it looks like this didn't fully get implemented from what I see but I think that's ok. I'm worried about this going too stale, so I've made an update here and if it passes the build I'll likely merge this and we can address this in a follow-up.